src/ecs/resources/mod.rs

```diff
@@ -11,6 +11,6 @@ mod broadcaster;
 
 pub use self::broadcaster::Broadcaster;
 pub use self::camera::{Camera, Projection};
-pub use self::input::{Axis, Button, InputHandler, PressedButtons, PressedKeys, PressedMouseButtons};
+pub use self::input::{Axis, Button, Buttons, KeyCodes, MouseButtons, InputHandler};
```

Comment:
So what do you suggest? I just think that alphabetical order helps when you read them (and it looks really clean and tidy).